pkg/driver_cryptocell/psa_cc/Kconfig

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2021 HAW Hamburg
```

Comment:
As discussed offline, personally I would like to harmonize the naming scheme a bit. There is "Crys", ""cc", "cc310", "cryptocell" all for the same thing.

Reply:
I renamed everything to `...cryptocell_310...` except for the CRYS stuff.
CRYS is a part of the library code and I can't change the way they chose to name their types and functions.